content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,133 @@
+# data status
+
+Show changes in the data tracked by DVC in the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+The `data status` command displays the state of the working directory and the
+committed changes with respect to the last Git commit (`HEAD`).
+
+It shows you what the new changes have been committed to DVC, which haven't been
+committed, which files aren't being tracked by DVC and Git, and what files are
+missing from the cache. It shows you what's been going on with `dvc add`,
+`dvc commit` and `dvc repro` commands.
+
+The `dvc data status` command only outputs information, it won't modify or
+change anything in your working directory. It's a good practice to check the
+state of your repository before doing `git commit` so that you don't
+accidentally commit something you don't mean to.
+
+An example output might look something like follows:
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc pull <file>..." to download files)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+        deleted: model.pkl
+(there are other changes not tracked by dvc, use "git status" to see)
+```
+
+As shown above, the `dvc data status` displays changes in multiple categories:
+
+- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
+  and `.dvc` files but the corresponding cache files are missing.
+- _DVC committed changes_ indicates that there are changes that are committed
```

Comment:
```suggestion
- _DVC committed changes_ indicates that there are changes that are `dvc-commit`-ed
```

For clarity and consistency with the next paragraph.

Reply:
Too many kinds of commit 😢